manifests/minimize_access.pp

```diff
@@ -47,6 +54,134 @@
       recurselimit => $recurselimit,
     }
   })
+# Added users with homes
+  $homes_users = split($::home_users, ',')
+
+# added ignore these homes
+  $target_home_users = difference($homes_users, $ignore_home_users)
+
+# added homes to restrict
+  ensure_resources ('file',
```

Comment:
As we don't want to break existing environments, those new features should be `opt-in` ... please add a flag to enable them (altogether or per feature)